code/drasil-docLang/lib/Drasil/Sections/Requirements.hs

```diff
@@ -41,18 +41,12 @@ reqF = SRS.require [reqIntro]
 -- For listing input requirements.
 fullReqs :: (Quantity i, MayHaveUnit i) => [i] -> Sentence -> [ConceptInstance] -> [ConceptInstance]
 fullReqs [] _ _ = []
-fullReqs i d r = nub $ inReq (inReqDesc (inTable i) d) : r-- ++ [outReq (outReqDesc outTable)]
+fullReqs i d r = nub $ inReq (inReqDesc (mkInputPropsTable i) d) : r-- ++ [outReq (outReqDesc outTable)]
 
 -- | Prepends given LabelledContent to an input-value table.
 fullTables :: (Quantity i, MayHaveUnit i) => [i] -> [LabelledContent] -> [LabelledContent]
 fullTables [] _ = []
-fullTables i t = inTable i : t
-
--- | Creates a generalized input-value table for the Requirements section.
-inTable :: (Quantity i, MayHaveUnit i) => [i] -> LabelledContent
-inTable i = mkInputPropsTable i (inReq EmptyS) -- passes empty Sentence to make stub of inReq
```

Comment:
Note: `inReq` is a chunk stub that creates an input FR with some given text ($X$) in the form `Input-Values: Input the values from Tab:ReqInputs, which define the X.`.

Note: we (ab)use `inReq` to create references to the input FR without referencing the specific input FR instantiated to an example. Any `inReq EmptyS` used to give a `UID` to the project's corresponding input FR is a cheat.